docker/importer/importer.py

```diff
@@ -217,7 +219,7 @@ def schedule_regular_updates(self, repo, source_repo):
 
     # Re-compute existing Bugs for a period of time, as upstream changes may
```

Comment:
```suggestion
    # Re-compute existing Bugs after a period of time, as upstream changes may
```

Reply:
I think this makes more sense to readers?

Reply:
I guess what's ambiguous here is what's happening after a period of time? The re-computing, or the selection of bugs being recomputed?

Reply:
Had a quick chat with Oliver and expanded the comment a bit, can you see if it's clearer now?